src/setup/config.ts

```diff
@@ -84,5 +87,14 @@ export function conf(): RuntimeConfig {
       .split(",")
       .map((v) => v.trim())
       .filter((v) => v.length > 0), // Should be comma-seperated and contain the media type and ID, formatted like so: movie-753342,movie-753342,movie-753342
+    CDN_REPLACEMENTS: getKey("CDN_REPLACEMENTS", "")
+      .split(",")
+      .map((v) =>
+        v
+          .split(":")
+          .map((s) => s.trim())
+          .filter((s) => s.length > 0),
+      )
+      .filter((v) => v.length > 0), // The format is <beforeA>:<afterA>,<beforeB>:<afterB>
```

Comment:
This should be checking if there are two elements in each array perhaps?

```suggestion
      .filter((v) => v.length === 2), // The format is <beforeA>:<afterA>,<beforeB>:<afterB>
```